src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.cs

```diff
@@ -417,6 +418,107 @@ class C
         }
 
         [Fact]
+        [WorkItem(217740, "https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_workitems?id=217740")]
+        public void DateTimeConstantAttribute()
+        {
+            #region "Source"
+            var source = @"
+using System;
+using System.Runtime.CompilerServices;
+
+public class Bar
+{
+    public void Method([DateTimeConstant(-1)]DateTime p1) { }
```

Comment:
Should test the same when default value is explicitly set through the special syntax on the parameter.
Should test similar scenarios for VB too. 

---
In reply to: [64497023](https://github.com/dotnet/roslyn/pull/11536#discussion_r64497023) [](ancestors = 64497023)